packages/components/src/components/Drawer/Drawer.tsx

```diff
@@ -1,25 +1,33 @@
 import React from "react";
 import { Dialog } from "ariakit/dialog";
 import type { DialogProps } from "ariakit";
+import { styled } from "@localyze-pluto/theme";
 import { Box } from "../../primitives/Box";
 
 export interface DrawerProps extends DialogProps {
   /** The contents of the drawer. */
   children: NonNullable<React.ReactNode>;
 }
 
+const StyledBackdrop = styled(Box.div)`
+  background-color: rgba(39, 49, 63, 0.5);
+  backdrop-filter: blur(3px);
+`;
+
 /** A Drawer is a page overlay that displays information and blocks interaction with the page until an action is taken or the Drawer is dismissed. */
 const Drawer = React.forwardRef<HTMLDivElement, DrawerProps>(
-  ({ children, ...props }, ref) => {
+  ({ children, initialFocusRef, ...props }, ref) => {
     return (
       <Box.div
         as={Dialog}
-        backdrop={false}
+        autoFocusOnShow={initialFocusRef ?? false}
+        backdrop={StyledBackdrop}
```

Comment:
> - Adds backdrop prop to Drawer component

from the description of the change I thought this would be an optional prop. Are we always going to have the backdrop? If so maybe we can update the message since it's actually not a prop of the component, but a prop that we pass to the div inside this component

> - Adds backdrop ~prop~ to Drawer component

Reply:
So you think the issue is with the changelog message?

Reply:
yeah maybe, when I read the message I thought it was going to be optional (that we could control it with a prop, and set the backdrop on or off)